es-vue-base/src/lib-components/EsFooter.vue

```diff
@@ -15,100 +15,111 @@
-                                    class="mt-25 mb-50">
-                                    <es-nav-bar-link
-                                        :href="link.url"
-                                        :target="link.newTab ? '_blank' : null"
-                                        class="text-reset">
-                                        {{ link.text }}
-                                    </es-nav-bar-link>
-                                </li>
-                            </ul>
+        <div class="bg-blue-800 py-200 px-200">
+            <div class="container px-0">
+                <div class="row">
+                    <!-- Link columns -->
+                    <div class="col col-12 col-lg-8 order-lg-1 text-white">
+                        <div class="row">
+                            <div
+                                v-for="column in content.columns"
+                                :key="column.title"
+                                class="col col-6 col-md-3 mb-150">
+                                <ul class="list-unstyled">
+                                    <li class="font-weight-bold font-size-75 font-size-md-100 mb-50">
+                                        {{ column.title }}
+                                    </li>
+                                    <li
+                                        v-for="link in column.links"
+                                        :key="link.text"
+                                        class="mt-25 mb-50 font-size-75">
+                                        <es-nav-bar-link
+                                            :href="link.url"
+                                            :target="link.newTab ? '_blank' : null"
+                                            class="text-reset">
+                                            {{ link.text }}
+                                        </es-nav-bar-link>
+                                    </li>
+                                </ul>
+                            </div>
                         </div>
-                    </div>
-                </div>  <!-- Link columns -->
-                <!-- Logo and Social-->
-                <div class="col col-12 col-lg-4 order-lg-0 mb-200">
-                    <es-nav-bar-link
-                        class="d-block mb-150"
-                        :href="content.home.link">
-                        <span class="sr-only">
-                            {{ content.home.name }}
-                        </span>
-                        <es-logo
-                            height="36px"
-                            width="160px" />
-                    </es-nav-bar-link>
-                    <!-- Social icons -->
-                    <div class="d-flex text-gray-700">
+                    </div>  <!-- Link columns -->
+                    <!-- Logo and Social-->
+                    <div class="col col-12 col-lg-4 order-lg-0 mb-200">
```

Comment:
On mobile, the logo and social links should be centered.

Reply:
Centering looks good.

But I see too much space below the social icons on mobile - this `mb-200` is combining with the `py-200` on a parent `<div>` for 64px of space, when it should just be 32px - maybe we remove the `mb-200` here? It looks like that won't negatively impact the desktop layout either, so should be okay to do.

<img width="386" alt="Screen Shot 2024-05-08 at 12 33 46 PM" src="https://github.com/EnergySage/es-ds/assets/1350363/972289b9-3a69-4cd2-87d8-596d463b7522">